config/default.json

```diff
@@ -52,5 +52,6 @@
       }
     }
   },
-  "apiCallTimeout" : 40000
+  "apiCallTimeout" : 40000,
+  "navigationTimeout" : 180000
```

Comment:
Is this not now the same as the mocha timeout? If so won't mocha timeout before this as we make multiple page requests per test?

Reply:
I mean theoretically yes :)  trying to figure out if this is the actual cause 180 sec is already too high for both anyway will decrease it to 140 sec just too see if this is the case...

Reply:
or we can also increase mocha time out even more but then I believe pipeline has a time limit of 40 min :)